Adjust assorted terminology and wording around product offerings. (#57242)
ansible/ansible: docs/docsite/rst/reference_appendices/tower.rst

.. _ansible_tower:

Red Hat Ansible Tower
=====================

`Red Hat Ansible Tower <https://www.ansible.com/products/tower>` is a web console and REST API for operationalizing Ansible across your team, organization, and enterpise. It's designed to be the hub for all of your automation tasks.

> Ansible Tower allows you to control access to who can access what through role-based access control, and allows sharing of securely stored credentials for SSH and other services.  Inventory can be synced with a wide variety of cloud sources, and powerful multi-playbook workflows allow you to model

Reviewer: ```suggestion
Ansible Tower gives you role-based access control, including control over the use of securely stored credentials for SSH and other services. You can sync your Ansible Tower inventory with a wide variety of cloud sources, and powerful multi-playbook workflows allow you to model
```

complex processes.

> It logs all of your jobs, integrates well with LDAP, SAML, and other account sources, and has an amazing browsable REST API. Command line tools are available for easy integration with Jenkins as well.

Reviewer: ```suggestion
It logs all of your jobs, integrates well with LDAP, SAML, and other authentication sources, and has an amazing browsable REST API. Command line tools are available for easy integration with Jenkins as well.
```

> Find out more about Ansible Tower features and how to download it on the `Ansible Tower webpage <https://www.ansible.com/products/tower>`_. Ansible Tower is part of the Red Hat Ansible Automation subscription, and comes bundled with amazing support from Ansible, Inc.

Reviewer: ```suggestion
Find out more about Ansible Tower features and how to download it on the `Ansible Tower webpage <https://www.ansible.com/products/tower>`_. Ansible Tower is part of the Red Hat Ansible Automation subscription, and comes bundled with amazing support from Red Hat, Inc.
```
